chpl/chpl-service/src/main/java/gov/healthit/chpl/dao/impl/DeveloperDAOImpl.java

```diff
@@ -376,6 +378,63 @@ public DeveloperDTO getByVersion(Long productVersionId) throws EntityRetrievalEx
 		DeveloperEntity ve = (DeveloperEntity)result;
 		return new DeveloperDTO(ve);
 	}
+	
+	public List<DeveloperDecertifiedDTO> getDecertifiedDevelopers(){
+		List<CertifiedProductDetailsEntity> result = entityManager.createQuery(
+				"FROM CertifiedProductDetailsEntity "
+				+"WHERE developerStatusName IN ('Suspended by ONC', 'Under certification ban by ONC') ", CertifiedProductDetailsEntity.class).getResultList();

```

Comment:
I have an enum gov.healthit.chpl.entity.DeveloperStatusType . Could you use that instead of typing the names in here? If they ever decide these status names need to be changed we might get lucky and only have to change them in one place in the code.